Sources/Interaction/Style/InteractorStyleImage/index.js

```diff
@@ -267,7 +257,7 @@ const DEFAULT_VALUES = {
   windowLevelCurrentPosition: [0, 0],
   lastSlicePosition: 0,
   windowLevelInitial: [1.0, 0.5],
-  currentImageProperty: 0,
+  // currentImageProperty: null,
```

Comment:
why commented & null?

Reply:
When you don't initialize a default value it is common practice to let it visible (so you know it is there) but there is no need to set its value to null as by default it will be undefined.